add working group `wg-key-visualizer` (#97)
pingcap/community: working-groups/wg-key-visualizer.md

# Key Visualizer Working Group

## Goals

Implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting and reasoning inefficient application usage patterns.

[RFC](https://github.com/tikv/rfcs/pull/367)

## Work and Meeting

* Regular Working Group Meeting: 4pm on every Thursday
* Meeting Zoom: [https://pingcap.zoom.com.cn/j/669899354](https://pingcap.zoom.com.cn/j/669899354)
* [Work Status & Materials](https://docs.google.com/document/d/1LdvS0oWo79eTINHCZAdI0z0vKGP2xiCLnHYKRHBwfh8)

## Mentors

> * [samyubowei](https://github.com/samyubowei), www.58.com

Reviewer: ```suggestion
* [samyubowei](https://github.com/samyubowei)([@www.58.com](www.58.com))
```

> * [nolouch](https://github.com/nolouch), [PingCAP](https://github.com/pingcap)

Reviewer: ditto

## Members

> * [sivagao](https://github.com/sivagao), [PingCAP](https://github.com/pingcap)

Reviewer: ditto

Author: done

# Key Visualizer Working Group

## Goals

> Implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting and reasoning inefficient application usage patterns.

Reviewer 2: ```suggestion
The Key Visualizer Working Group aims to implement a visual component to show key heatmap of the TiKV cluster, which is useful for troubleshooting inefficient application usage patterns.
```

[RFC](https://github.com/tikv/rfcs/pull/367)

## Work and Meeting

* Regular Working Group Meeting: 4pm on every Thursday
* Meeting Zoom: [https://pingcap.zoom.com.cn/j/669899354](https://pingcap.zoom.com.cn/j/669899354)
* [Work Status & Materials](https://docs.google.com/document/d/1LdvS0oWo79eTINHCZAdI0z0vKGP2xiCLnHYKRHBwfh8)

## Mentors

* [samyubowei](https://github.com/samyubowei)([@58.com](www.58.com))

> * [nolouch](https://github.com/nolouch), [PingCAP](https://github.com/pingcap)

Reviewer: ```suggestion
* [nolouch](https://github.com/nolouch), [@PingCAP](https://github.com/pingcap)
```

## Members

> * [sivagao](https://github.com/sivagao), [PingCAP](https://github.com/pingcap)

Reviewer: ```suggestion
* [sivagao](https://github.com/sivagao), [@PingCAP](https://github.com/pingcap)
```

> * [andylokandy](https://github.com/andylokandy), [PingCAP](https://github.com/pingcap)

Reviewer: ```suggestion
* [andylokandy](https://github.com/andylokandy), [@PingCAP](https://github.com/pingcap)
```

> * [hundundm](https://github.com/hundundm), [PingCAP](https://github.com/pingcap)

Reviewer: ```suggestion
* [hundundm](https://github.com/hundundm), [@PingCAP](https://github.com/pingcap)
```

> * [zzh-wisdom](https://github.com/zzh-wisdom), [HUST](http://www.hust.edu.cn/)

Reviewer: ```suggestion
* [zzh-wisdom](https://github.com/zzh-wisdom), [@HUST](http://www.hust.edu.cn/)
```

> * [Sullivan12138](https://github.com/Sullivan12138), [HUST](http://www.hust.edu.cn/)

Reviewer: ```suggestion
* [Sullivan12138](https://github.com/Sullivan12138), [@HUST](http://www.hust.edu.cn/)
```
